src/routes/Dashboard/index.js

```diff
@@ -1,12 +1,19 @@
+// React Imports
+import { useEffect, useState } from "react";
+
+// Material UI Imports
 import Button from "@mui/material/Button";
+import { Typography } from "@mui/material";
+
+// Component Imports
 import TaskList from "./TaskList";
-import { useEffect, useState } from "react";
 import TaskForm from "./TaskForm";
 import TaskEditForm from "./TaskEditForm";
-import { Typography } from "@mui/material";
-import axios from "../../api/axios";
 import Spinner from "../../components/Spinner";
 
+// API Imports
+import { getTodos, getTodo, postTodo, putTodo, deleteTodo } from '../../api';
```

Comment:
From previous comment, one suggestion instead of this: `import * as api from '../../api';`

Reply:
Good suggestion. I'll use this moving forward.